dockstore-webservice/src/main/java/io/dockstore/webservice/CustomWebApplicationException.java

```diff
@@ -31,6 +31,11 @@ public CustomWebApplicationException(String message, int status) {
         this.errorMessage = message;
     }
 
+    public CustomWebApplicationException(String message, int status, Throwable ex) {
```

Comment:
Not sure you need this (see my other comment)

Reply:
Removed overloaded constructor with third parameter, since no longer passing the Throwable into the custom exception.